public/index.html

```diff
@@ -11,9 +11,14 @@
       integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
       crossorigin="anonymous"
     />
+    <style>
+      body {
+        background-color: black !important;
```

Comment:
when we turn on CSS linting, !important will be banned..

Reply:
Yea, it's a temp fix because bootstrap is imported as the last head item by default

Reply:
![source](https://user-images.githubusercontent.com/3502798/111617395-f4866200-87da-11eb-9ba8-f6a471fd1411.gif)